README.md

```diff
@@ -53,3 +53,15 @@ A smarter mouse trap with SMS alerting.
 </p>
 
 ---
+
+
+### 6. 🅿️ Parking Lot Help Request
```

Comment:
lol at 6 being in here before 3 and 4 😛 

We might have to talk to Brandon about a potential reorder.

Reply:
Probably so